runtime/testy/runtime.py

```diff
@@ -86,7 +86,7 @@ def nonTestModePrint(*args):
           spawnProcess(PROCESS_NAMES.UDP_RECEIVE_PROCESS, startUDPReceiver)
           dawn_connected = False
           controlState = "idle"
-          break
+          continue
```

Comment:
I think this should be break, since we want to kill studentCode if Dawn disconnects. @bodiddles ?